### PR TITLE
Add rockylinux8 containers

### PR DESCRIPTION
**Pull request by amanda11**
Containers for build and test with rocky8.

Part of https://github.com/StackStorm/community/issues/88

### CODE REVIEW

**Comment by amanda11** [comment]
Differences to centos8:

- Had to install procps so that st2ctl would work - so ps would work (possibly should move out of test image and into the installers?)
- Had to install iproute - so that had utility ss - otherwise the rake test on the st2-packages would fail.

**Comment by amanda11** [comment]
Successfully ran st2-packages using the rockylinux containers to build and test st2 rpm.
Keeping in draft, until managed to build st2web and st2chatops rpms and also tested to see compatabiliy of those RPMs with other EL8 distros.

File: packagingbuild/rockylinux8/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,98 @@
+FROM rockylinux:8
+
+# Download tools
+RUN yum -y install \
+    ca-certificates \
+    curl \
+    wget
+
+RUN yum -y install \git \
```
Comment (stevemuskiewicz) [inline packagingbuild/rockylinux8/Dockerfile:9]: misplaced backslash before `git`?

File: packagingbuild/rockylinux8/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    mercurial \
+    openssh \
+    subversion
```
Comment (stevemuskiewicz) [inline packagingbuild/rockylinux8/Dockerfile:12]: Just curious, are mercurial and subversion really still used here? 🙂

Comment (amanda11) [reply packagingbuild/rockylinux8/Dockerfile]: @stevemuskiewicz Removed as weren't needed, and yes didn't need the \ above. Thanks for the review comments.

**Review by armab — APPROVED** [review]
:+1: 

You'll need to build and push the new rockylinux image manually though, as Docker Hub removed the free builds.

**Pull request merged**